docs/Documentation/tutorial.md

```diff
@@ -1,23 +1,25 @@
 # Tutorial for Running the VIC-RBM hydrologic and stream temperature model
-These are the instructions for running the integrated modeling system comprised of the large-scale hydrologic model, VIC ([Liang et al, 1994](../Documentation/References.md)), a routing model based on the work of [Lohmann et al (1996)](../Documentation/References.md) and the semi-Lagrangian water temperature model, RBM ([Yearsley, 2009,2012](../Documentation/References.md)). Model development and implementation for the large-scale hydrologic model, VIC are described in detail on the Web site of the University of Washington Department of Civil and Environmental Engineering's [UW Hydro|Computational Hydrology group](https://uw-hydro.github.io/code/)
+These are the instructions for running the integrated modeling system comprised of the large-scale hydrologic model **VIC** ([Liang et al, 1994](../Documentation/References.md)), a routing model based on the work of [Lohmann et al (1996)](../Documentation/References.md) and the semi-Lagrangian water temperature model, RBM ([Yearsley, 2009,2012](../Documentation/References.md)). Model development and implementation for the large-scale hydrologic model **VIC** are described in detail on the website of the University of Washington Department of Civil and Environmental Engineering's [UW Hydro|Computational Hydrology group](https://uw-hydro.github.io/code/)
```

Comment:
Fixed first one! As for second, I prefer to keep it there cause this is a prorogation of our group ;)